mautrix/client/api/user_data.py

```diff
@@ -170,3 +170,16 @@ async def get_profile(self, user_id: UserID) -> Member:
             raise MatrixResponseError("Invalid member in response") from e
 
     # endregion
+
+    # region Beeper Custom Fields API
+
+    async def beeper_update_profile(self, custom_fields: dict) -> None:
```

Comment:
```suggestion
    async def beeper_update_profile(self, custom_fields: dict[str, Any]) -> None:
```

(need to import `Any`)